config/horizon.json

```diff
@@ -91,6 +91,7 @@
 		"server-side-rendering": true,
 		"settings/security/monitor": true,
 		"settings/theme-setup": false,
+		"sign-in-with-apple": false,
```

Comment:
We can probably have it working on horizon IF we keep using the popup flow only (we're limited in the amount of redirect uri we can have for SIWA)